client/my-sites/themes/themes-magic-search-card/welcome.jsx

```diff
@@ -96,7 +114,10 @@ class MagicSearchWelcome extends React.Component {
 					className="themes-magic-search-card__welcome-taxonomy-icon"
 					size={ 18 }
 				/>
-				{ taxonomy }
+				{ taxonomyTranslations[ taxonomy ] ||
+					i18n.translate( 'Unknown', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **102** times:
`translate( 'Unknown', { context: 'name'} )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/theme-showcase-filter-translations)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?